src/actinia_core/version.py

```diff
@@ -133,3 +135,20 @@ def version():
     info['running_since'] = find_running_since_info()
 
     return make_response(jsonify(info), 200)
+
+
+# Return a hint that this version is outdated
+@flask_app.route("/api/v1/<path:actinia_path>")
+def hint(actinia_path):
+    """Return a hint that this version is no longer installed. If an older
+    version is installed, this endpoint will be overwritten by a proxy.
+
+    Returns: Response
+
+    """
+    url = request.url_root.strip('/') + URL_PREFIX + '/' + actinia_path
+
+    return make_response(jsonify(LinkResponseModel(
+            status="Not found",
+            message="Are you looking for the current api version?",
```

Comment:
Maybe it helps to specify in the message which part of the URL can be adapted. Since so far it had always been `.../api/v1/...`, it might not occur to everyone that it is the `v1` that needs to be changed (Although quite obvious, but you never know ;))

Reply:
yes, from a beginner's point of view I would definitely agree, that I probably wouldn't get it right away...

Reply:
Ok, I changed it in the description above accordingly.